<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>1001</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Security Voice code Access</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,8,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,1">
      <item>
       <widget class="QTableWidget" name="similarity_table">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>240</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>81</number>
        </attribute>
        <row>
         <property name="text">
          <string>Open The Door</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Unlock Middle Gate</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Give Me Access</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>mariamH</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>mariamM</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>ali</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>mina</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>ahmed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>hazem</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>nourhan</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>mayar</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Similarity</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0%</string>
         </property>
         <property name="flags">
          <set>ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0%</string>
         </property>
         <property name="flags">
          <set>ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>0%</string>
         </property>
         <property name="flags">
          <set>ItemIsEditable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="5" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="6" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="7" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="8" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="9" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
        <item row="10" column="0">
         <property name="text">
          <string>0%</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="mode_group_box">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Select Your Mode</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QRadioButton" name="mode_1_radiobtn">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mode 1 – Security voice code</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QRadioButton" name="mode_2_radiobtn">
             <property name="text">
              <string>Mode 2 – Security voice fingerprint</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="spectrogram_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,3,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="access_label">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
color : red;
}</string>
            </property>
            <property name="text">
             <string>Access Denied</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>32</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="access_icon">
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>600</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>icons/padlock_2886699.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>33</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="mic_button">
            <property name="minimumSize">
             <size>
              <width>302</width>
              <height>302</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>302</width>
              <height>302</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{ 
background-color: white;
 border-style: solid;
 border-width:1px;
 border-radius:150px;
 border-color: red;
 max-width:300px;
 max-height:300px;
 min-width:300px;
 min-height:300px;
}
QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/1082810.png</normaloff>icons/1082810.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1036</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
